tv.html

```diff
@@ -0,0 +1,239 @@
+      <div class="latest-message">
+        <div class="latest-message-background soft-half-bottom push-bottom">
+          <div class="embed-container">
+            <div class="media-video">
+              <div class="card-video-player">
+                <div class="embed-responsive embed-responsive-16by9" id="js-media-video" player-id="player"
+                  video-id="9vmwUS1E-Lc" data-video-player>
+                  <div id="player"></div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <h2 class="latest-message-headline font-family-condensed-extra text-uppercase flush-top text-gray-dark">
+            Is God Good? | The Good God Journey Week 1
+          </h2>
+        </div>
+      </div>
+    </div>
+    <div class="col-sm-4">
+      <div class=" bg-charcoal media featured soft-sides">
+        <div class="media-body soft-bottom">
+          <h2 class="text-uppercase font-family-serif font-size-base font-size-small text-white">current series</h2>
+          <a href="https://www.crossroads.net/media/series/the-good-god-journey" class="push-half-bottom"
+            data-automation-id="series-image">
+
+            <img class="full-width"
+              src="https://crds-media.imgix.net/5a5Jw5jhgnTeSjdyHejpWO/791786a39157f100723274cadc0bc73a/journey2020-good-god.jpg?{{ site.imgix_params.placeholder_sixteen_nine }}"
+              sizes="{{ site.image_sizes.one_fourth }}" data-optimize-img />
+          </a>
+          <h5 class="component-header font-size-h3 text-white">The Good God! Journey</h5>
+          <div class="soft-quarter-ends font-size-smaller push-bottom text-white">
+            2020 has been hard on everyone. A journey is a chance to reconnect, learn, and grow together. The Journey
+            might just be the best thing you do all year.
+          </div>
+          <crds-button href="https://www.crossroads.net/media/series/the-good-god-journey" color="white"
+            display="outline" text="View the series" />
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="bg-tan-light soft-ends">
+  <div class="container soft-bottom">
+    <div class="latest-message">
+      <hr class="flush-top push-half-bottom push-half-sides" />
+      <div class="latest-message-background soft-half-bottom push-bottom">
+        <h2
+          class="text-uppercase font-family-serif font-size-base font-size-small text-gray-light flush-top push-half-sides">
+          real encounters with God
+        </h2>
+        <div class="col-sm-4">
+          <div class="embed-container">
+            <a href="https://www.crossroads.net/media/series/real-encounters-with-god">
+              <div class="media-video">
+                <img class="media-img full-width"
+                  src="https://crds-media.imgix.net/4290QD6wnvVOL98cPnlmwz/0c6e9d0e07f266a32e0bbcc7690c42e2/Screen_Shot_2020-03-06_at_4.05.46_PM.png?auto=format,compress&w=750&h=422&fit=crop"
```

Comment:
drop the `https:`

Reply:
Also why are you setting the width of the image in the url and have a class of `full-width` on the img?